### PR TITLE
Put the inputs and button in a list

### PR DESCRIPTION
**Pull request by lolashot**
**Contact-Form Update**

1. Create Html $ CSS Files
2. Create Readme
3. Create Assets
4. Add the Linter folder

Contact form added to my portfolio with form validation and data collection mechanism after submitting the form.

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @lolashot  , 
### I'm Levy, your PR reviewer

Excellent, you did a great job :sunglasses:
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Good job for using the correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Descriptive README :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 
_check comments under_
- [ ] Kindly make sure to place your form in the center of the contact form section in order to stick to the provided Figma design. :wink: 
   - ### Your design
   image

   - ### Figma design
  
   image

- [ ] Kindly adjust the margins of the contact section form because it took more than 100% width of that page. :wink: 

   - ### Your design
  image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -361,25 +372,28 @@ h3 {
...
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
   color: #fff;
-  padding-top: 60px;
-  letter-spacing: 0em;
+  padding-top: 90px;
+  letter-spacing: 0;
   margin-bottom: 12px;
```
Comment (franclobo) [inline style.css:383]: - [ ] Kindly fix the margins of contactTitle to look similar to the Figma template. 👇🏽 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

image

File: style.css [hunk 2]
```diff
@@ -361,25 +372,28 @@ h3 {
...
 
 .contactText {
-  font-family: 'Poppins', Helvetica, Arial, Verdana, Tahoma, sans-serif;
+  font-family: "Poppins", Helvetica, Arial, Verdana, Tahoma, sans-serif;
   font-style: normal;
-  font-weight: 400;
+  font-weight: 300;
   font-size: 20px;
   line-height: 24px;
   color: #ebebff;
```
Comment (franclobo) [inline style.css:392]: - [ ] Plase 🙏🏽 fix the margins of the contactText to look similar to the Figma template.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi, @MILINDI7 and @lolashot, 

You've done it well, congratulations!! 😃 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR 💯 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]** Add your partner information in the authors' section on the README file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @lolashot  👋,

_This is an additional review as a result of the second opinion request._

Kudos for the nice project you implemented!

I agree that you fixed the issue spotted by the previous reviewer and that your project is ready to be approved.

I will just suggest you some optional requirements:
- my opinion is that in order to take full advantage of the Flexbox and Grid, is better to avoid using fixed widths and heights.

Please take a look at how your UI looks on different mobile view sizes:
https://www.loom.com/share/2db0c75fe8184db498243958cf9ba9dc

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @lolashot       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**